packages/trax/trax.0.3.0/descr

```diff
@@ -0,0 +1 @@
+Trax defines a special exception, which is used to store a trace of where the exception was raised and re-raised, independently from the state of the call stack.
```

Comment:
Thanks. The description should look better now.

I still don't understand Github's review process. Am I supposed to do something to let you know that I'm done with my changes or do you have to guess?